AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/States/DeviceSRPState.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AWSCognitoIdentityProvider
+
+enum DeviceSRPState: State {
+
+    case notStarted
+    case initiatingDeviceSRPA(SRPStateData)
```

Comment:
We could drop the 'A' 
```suggestion
    case initiatingDeviceSRP(SRPStateData)
```